docs/UserGuide.md

```diff
@@ -44,11 +44,10 @@ It allows for faster and more effective student management.
   * `add_student n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 l/Sec2 g/95421323 r/Mother`: Adds a student John Doe to the Tutor Buddy application
   * `delete_student 3`: Deletes the 3rd student in the student list
 
-   **Tuition Session**
+    **Tuition Session**
   * `list_session`: List all tuition sessions
-  * `find_session James`: Finds and lists all tuition sessions that James have
```

Comment:
Should we comment out find_session here instead of deleting them?

Reply:
Commenting out in the middle of list items creates a bit of a weird gap in between. I'll take your suggestion and comment it out at the bottom, so we can restore it easier next time!

Reply:
Great!